mentoris/settings.py

```diff
@@ -81,13 +81,13 @@
 
         'NAME': 'mentoris',
 
-        'USER': 'walden',
+        'USER': 'ashton',
```

Comment:
We don't want your personal DB credentials pushed to main. Revert back to the default walden user and submit another PR and we can probably push your changes.